terraform/modules/stack/iam_policy_document.tf

```
@@ -133,3 +135,58 @@ data "aws_iam_policy_document" "archivematica_ingests_get" {
     ]
   }
 }
+
+// This is the default policy that gets added to a topic when no policy is supplied
+// plus the statement that allows other principals to subscribe to this topic
+data "aws_iam_policy_document" "register_output_subscribe" {
+  policy_id = "__default_policy_ID"
+
+  // default permissions copied from https://www.terraform.io/docs/providers/aws/r/sns_topic_policy.html
+  statement {
+    actions = [
+      "SNS:Subscribe",
+      "SNS:SetTopicAttributes",
+      "SNS:RemovePermission",
+      "SNS:Receive",
+      "SNS:Publish",
+      "SNS:ListSubscriptionsByTopic",
+      "SNS:GetTopicAttributes",
+      "SNS:DeleteTopic",
+      "SNS:AddPermission",
+    ]
+
+    condition {
+      test     = "StringEquals"
+      variable = "AWS:SourceOwner"
+
+      values = [
+        "${data.aws_caller_identity.current.account_id}",
```

nit: in terraform 0.12, we get to drop the quotes around things, and that's what we've been doing
```suggestion
        data.aws_caller_identity.current.account_id,
```